nautobot/extras/filters.py

```diff
@@ -435,6 +437,35 @@ def search(self, queryset, name, value):
         )
 
 
+#
+# Dynamic Groups
+#
+
+
+class DynamicGroupFilterSet(
+    BaseFilterSet,
+    CustomFieldModelFilterSet,
+    CreatedUpdatedFilterSet,
+):
```

Comment:
Ideally:

```suggestion
class DynamicGroupFilterSet(NautobotFilterSet):
```

though this will require some moving around of code since `NautobotFilterSet` is defined below at line 467/498.

Not blocking!